### PR TITLE
Add blurred text shadow; Fix angle and distance never getting set to 0

### PR DESCRIPTION
**Pull request by onion-soup**
* Adds the canvas' `shadowBlur` feature in order to render blurred shadow behind the text.
* Fixes the constructor so that we can set `dropShadowAngle` and `dropShadowDistance` to `0`. Before it would just overwrite it with the default values.

### CODE REVIEW

File: src/core/text/Text.js [hunk 1]
```diff
@@ -196,8 +198,9 @@ Object.defineProperties(Text.prototype, {
 
             style.dropShadow = style.dropShadow || false;
             style.dropShadowColor = style.dropShadowColor || '#000000';
-            style.dropShadowAngle = style.dropShadowAngle || Math.PI / 6;
-            style.dropShadowDistance = style.dropShadowDistance || 5;
+            style.dropShadowAngle = style.dropShadowAngle !== undefined ? style.dropShadowAngle : Math.PI / 6;
+            style.dropShadowDistance = style.dropShadowDistance !== undefined ? style.dropShadowDistance : 5;
+            style.dropShadowBlur = style.dropShadowBlur !== undefined ? style.dropShadowBlur : 0; //shadowBlur is '0' by default according to HTML
```
Comment (drkibitz) [inline src/core/text/Text.js]: Probably not really part of this PR, but since I just saw this, thought I would comment that it would be great if this just used `Object.assign` here:
```js
this._style = Object.assign({}, Text.defaultStyle, style);
```
Yes, `Text.defaultStyle` does not exist just yet, but you get the idea.

Comment (GoodBoyDigital) [inline src/core/text/Text.js]: Thats a great idea @drkibitz !

**Comment by GoodBoyDigital** [comment]
Thanks for this pr @onion-soup ! I think we could definitely look into @drkibitz  idea about using object.assign when we get some time too!
